### ISSUE TITLE
SwahBERT Fine tuning

### ISSUE THREAD

**Issue by metabloit-m** [gatimartin/SwahBERT#1]
I followed the documentation: [SwahBERT: Language Model of Swahili](https://aclanthology.org/2022.naacl-main.23) (Martin et al., NAACL 2022) and was able to obtain the swahBERT model. 

My comfort zone is tensorflow so I had to convert the model (uncased) to a tensorflow model (using onnx). The conversion was successful and was able to run the model on tensorflow. The issue comes in the downstream task where I tested the model specifically for the emotional classification task but I get an accuracy of 50% (based on the test set, haven't computed F1 score as of yet). Regarding the architecture, I added a linear layer of size 64 and a dropout layer (0.1), and lastly the classifier layer with a sigmoid function. As per the research paper, it seems the model performed better based on the F1 score. How can I achieve this? Also, what parameters did you use during model compilation? And what layers (together with their parameters) were added during this task.

Also, is it that the learning rate in pytorch different from the one in tensorflow?
